go/datas/database_server.go

```diff
@@ -56,7 +57,7 @@ func (s *RemoteDatabaseServer) Run() {
 	d.Chk.NoError(err)
 	s.port, err = strconv.Atoi(port)
 	d.Chk.NoError(err)
-	fmt.Printf("Listening on port %d...\n", s.port)
+	log.Printf("Listening on port %d...\n", s.port)
```

Comment:
I'm a bit confused...above, you changed `fmt.Printf` to `verbose.Log`, while here you changed it to `log.Printf`